privacy-protections/runtime-checks/pages/test1.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Runtime checks</title>
+  <link rel="stylesheet" href="../style.css">
+</head>
+<body>
+    <script src="../utils.js"></script>
+    <p><a href="../index.html">[Runtime checks]</a></p>
+
+    <p>This page verifies that runtime checking is enabled given the corresponding <a href="../config/test1.json">config</a></p>
```

Comment:
Could we expand those text descriptions to say what this is testing and what's the expected outcome?